prow/scripts/e2e-test/template.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
same for this template.json. Do we need this in test-infra repo as well? We have this in reconciler repo, which we are using in the script.

Reply:
removed